image/image.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Images</title>
+    <link rel="stylesheet" href="image.css">
+</head>
+<body>
+    <div class="image-container">
+        <img src="https://images.unsplash.com/photo-1468877294001-94aef5ebfa1e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+        class="img"
+        />
+<br/>
+<br/>
```

Comment:
Try to avoid using <br> tag, as it is not good practice.

Reply:
done. thank you.